azkaban-web-server/src/web/js/azkaban/view/job-edit.js

```diff
@@ -86,12 +86,14 @@ azkaban.JobEditView = Backbone.View.extend({
       mythis.overrideParams = overrideParams;
       mythis.generalParams = generalParams;
 
-      for (var okey in overrideParams) {
-        if (okey != 'type' && okey != 'dependencies') {
-          var row = handleAddRow();
-          var td = $(row).find('span');
-          $(td[0]).text(okey);
-          $(td[1]).text(overrideParams[okey]);
+      if (overrideParams && $(".editRow").length == 0)  {
```

Comment:
Based on what I understand, your fix in `flow-execute-dialog.js` seems to fix the flow parameter display issue in #1481. I'm wondering what this change is for. Does Job property Editing have similar issue?

Reply:
Ah right! I forgot to mention it: yes, Job Editing had the same issue, I discovered it while checking the usages of the `. editRow` class. Since I was on it, I fixed it as well.

Reply:
Thanks! Please fill in some description in the description panel, which will serve as the commit message.

Reply:
Um, I tried but failed to reproduce the job editing bug. What scenario will generage this issue?

Reply:
Apparently the bug appears only if you close the dialog by clicking outside of it; if you hit `Cancel` instead, the properties correctly appear only once.

Reply:
be able to reproduce! Thanks for repairing! Could you please fill in some description? Then we are good to merge.